lang/java/reef-runtime-azbatch/src/main/java/org/apache/reef/runime/azbatch/util/RemoteIdentifierParser.java

```diff
@@ -0,0 +1,41 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+package org.apache.reef.runime.azbatch.util;
+
+/**
+ * Utility class to parse IP and port information from Azure Batch Node ID.
+ */
+public final class RemoteIdentifierParser {
+
+  private static final String PROTOCOL = "socket://";
```

Comment:
Could you add an example of what is a sample string that will be parsed with this class?
Also, should there be any exception handling if the input string does not meet the required criteria?